docker-compose.env.example

```diff
@@ -1,4 +1,8 @@
 ROLLBAR_ACCESS_TOKEN=ROLLBAR_ACCESS_TOKEN
 ROLLBAR_ENV=development
 
-DATABASE_URL=postgres://postgres@localhost:5432/roda-development
\ No newline at end of file
+DATABASE_URL=postgres://postgres@localhost:5432/roda-development
```

Comment:
Since [we've chosen to not use docker in dev and to bin Docker Compose](https://dxw.slack.com/archives/CN0H2L066/p1572541650015200) we can remove this, and move the example keys into .env.example I think?